translator/config/mode.go

```diff
@@ -10,6 +10,10 @@ const (
 	ModeWithIRSA  = "withIRSA"
 )
 
+const (
+	ModeECS = "ECS"
```

Comment:
If this is only going to be used in the EMF translator, then keep it there. Want to make sure we don't accidentally set the translator context to ECS since that would break a lot of things.

Reply:
It's used in EMF and applicationsignals translator.